lib-ui-compose/src/androidMain/kotlin/com/ustadmobile/libuicompose/util/passkey/GetCredentialUseCaseImpl.kt

```diff
@@ -0,0 +1,91 @@
+package com.ustadmobile.libuicompose.util.passkey
+
+import android.content.Context
+import android.util.Base64
+import androidx.credentials.CredentialManager
+import androidx.credentials.GetCredentialRequest
+import androidx.credentials.GetPasswordOption
+import androidx.credentials.GetPublicKeyCredentialOption
+import androidx.credentials.PasswordCredential
+import androidx.credentials.PublicKeyCredential
+import androidx.credentials.exceptions.GetCredentialException
+import androidx.credentials.exceptions.NoCredentialException
+import com.ustadmobile.core.domain.passkey.CredentialResult
+import com.ustadmobile.core.domain.passkey.GetCredentialUseCase
+import com.ustadmobile.core.domain.passkey.PassKeySignInData
+import com.ustadmobile.core.domain.passkey.PasskeyRequestJsonUseCase
+import org.json.JSONObject
+
+class GetCredentialUseCaseImpl(
+    val context: Context,
+    val passkeyRequestJsonUseCase: PasskeyRequestJsonUseCase
+) : GetCredentialUseCase {
+    override suspend fun invoke(domain: String): CredentialResult {
+        val credentialManager = CredentialManager.create(context)
+
+        val getPasswordOption = GetPasswordOption()
```

Comment:
Get Password option needs parameters to ensure that we only show passwords for the given learning space url.